.github/workflows/cluster_endtoend_onlineddl_revert_mysql80.yml

```diff
@@ -49,28 +38,28 @@ jobs:
             - 'tools/**'
             - 'config/**'
             - 'bootstrap.sh'
-            - '.github/workflows/cluster_endtoend_vtgate_partial_keyspace.yml'
+            - '.github/workflows/**'
 
     - name: Set up Go
-      if: steps.skip-workflow.outputs.skip-workflow == 'false' && steps.changes.outputs.end_to_end == 'true'
-      uses: actions/setup-go@v3
+      if: steps.changes.outputs.end_to_end == 'true'
+      uses: actions/setup-go@v2
       with:
-        go-version: 1.19.3
```

Comment:
@maksimov these backports change `go-version` from `1.19.3` -> `1.18.7`. Were these changes you added and do we need to keep them? 🙇

Reply:
I believe we need to keep it at 1.19.3+ for security vulnerabilities, but it's better to double check with Stas once he's back.